app/app/management/commands/sync_orgs_repos.py

```diff
@@ -0,0 +1,113 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User, Group
+from dashboard.models import Profile, Organization
+from git.utils import get_organization, get_user, get_repo
+from app.utils import sync_profile
+
+
+class Command(BaseCommand):
+    help = 'Synchronizes Organizations and Repo roles to members'
+
+    # def add_arguments(self, parser):
+
+    def handle(self, *args, **options):
+
+        try:
+            print("Loading Users....")
+            all_users = User.objects.all()  # potentially want to add some filters here around what users are synced
```

Comment:
What filters are you thinking?

Reply:
inactive users, or some other key that would be used to determine what users we shoudl sync, potentially an argument